parsers/CA_QC.py

```diff
@@ -0,0 +1,131 @@
+from datetime import timedelta, timezone
+import logging
+
+# The arrow library is used to handle datetimes
+#import arrow
+
+# The request library is used to fetch content through HTTP
+import requests
+# pandas processes tabular data
+#import pandas as pd
+
+from pprint import pprint
+import json
+
+PRODUCTION_URL = "https://www.hydroquebec.com/data/documents-donnees/donnees-ouvertes/json/production.json"
+CONSUMPTION_URL = "https://www.hydroquebec.com/data/documents-donnees/donnees-ouvertes/json/demande.json"
+
+
+def fetch_production(
+    zone_key="CA-QC",
+    session=None,
+    target_datetime=None,
+    logger=logging.getLogger(__name__),
+) -> dict:
+    """Requests the last known production mix (in MW) of a given region.
+       In this particular case, translated mapping of JSON keys are also required"""
+
+    def if_exists(elem: dict, etype: str):
+
+        english = {
+            "hydraulique": "hydro",
+            "thermique": "thermal",
+            "solaire": "solar",
+            "eolien": "wind",
+            "autres": "unknown",
+            "valeurs": "values",
+        }
+        english = {v: k for k, v in english.items()}
+        try:
+            return elem["valeurs"][english[etype]]
+        except KeyError:
+            return 0.0
+
+    data = _fetch_quebec_production()
+    for elem in reversed(data["details"]):
+        if elem["valeurs"]["total"] != 0:
+
+            return {
+                "zoneKey": zone_key,
+                "datetime": elem["date"],
```

Comment:
Currently, running `test-parser CA-QC` fails due to the dates:
`Datetimes must be returned as native datetime.datetime objects`

We can fix this by using arrow like this :)
```suggestion
                "datetime": arrow.get(elem["date"]).datetime,
```